libnopegl/src/backends/vk/gpu_ctx_vk.c

```diff
@@ -1318,7 +1318,7 @@ static void vk_begin_render_pass(struct gpu_ctx *s, struct rendertarget *rt)
             .extent.width  = rt->width,
             .extent.height = rt->height,
         },
-        .clearValueCount = rt_vk->nb_clear_values,
+        .clearValueCount = (uint32_t)rt_vk->nb_clear_values,
```

Comment:
Same question here (size_t vs uint32_t).

Reply:
Same answer wrt dereferencing `clear_values`. That being said it doesn't seem to happen for this one yet, so the rationale was simply consistency.

Reply:
Done